app/assets/stylesheets/homes.scss

```diff
@@ -31,15 +31,18 @@ body {
 	clear: both;
 }
 
-a {
-	color: $dark;
-	text-decoration: none;
-	outline: none;
-}
+.container {
 
-a:hover,
-a:focus {
-	color: $white;
+  a {
+  	color: $dark;
+  	text-decoration: none;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered color, outline, text-decoration